src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss

```diff
@@ -123,36 +123,57 @@ $round-image-width: 220px;
   font-weight: 500;
 }
 
+.subInfoWrapper {
+  @include shadow-inset-bottom;
+
+  background-color: $color-primary--020;
+}
+
 .subInfoContainer {
   @include container;
 
-  display: flex;
-  flex-direction: column;
   padding-top: 42px;
   padding-bottom: 42px;
+}
 
-  @include for-tablet-portrait-up {
-    flex-direction: row;
-  }
+.legend {
+  margin: 0 0 12px;
+  font-size: 22px;
+  font-weight: 300;
 }
 
-.subInfoTextWrapper {
-  flex: auto;
+.legendRow {
   margin-bottom: 32px;
 
-  @include for-tablet-portrait-up {
+  &:last-child {
     margin-bottom: 0;
-    margin-right: 64px;
+  }
+
+  @include for-tablet-landscape-up {
+    display: flex;
+    margin-bottom: 10px;
+  }
+}
+
+.icon {
+  margin-right: 14px;
+}
+
+.term {
+  font-weight: 500;
+  margin-bottom: 10px;
+
+  @include for-tablet-landscape-up {
+    flex-shrink: 0;
```

Comment:
I think for the benefit of those less familiar with flexbox, `flex: none;` is much clearer. You should almost never need to set flex-shrink (or basis, or grow) directly (spec recommends using the shorthand syntax instead). Do you know about [the shorthands](https://www.w3.org/TR/css-flexbox-1/#flex-common) for `flex`? They should cover everything.

Reply:
I sometimes use shorthand in the form `flex: x x x` which I thought set shrink, grow and basis without explicitly saying so, which I had thought was actually *more* confusing than listing them each out individually.